js_modules/dagster-ui/packages/ui-core/src/instigation/TickHistory.tsx

```diff
@@ -194,13 +194,6 @@ export const TicksTable = ({
 
   return (
     <>
-      {logTick ? (
-        <TickLogDialog
-          tick={logTick}
-          instigationSelector={instigationSelector}
-          onClose={() => setLogTick(undefined)}
-        />
-      ) : null}
```

Comment:
Looks like this isn't actually used! I'll fix it so that the detail dialog and log dialog are both top-level.